Patches/MechanoidsExtraordinaire/HediffDefs/Hediffs_Local_AddedParts_MechanoidsExtraordinaire.xml

```diff
@@ -8,15 +8,15 @@
 				<modName>Mechanoids Extraordinaire</modName>
 			</li>
 			<li Class="PatchOperationReplace">
-				<xpath>Defs/HediffDef[defName="MechaTentacle"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/verbs</xpath>
+				<xpath>Defs/HediffDef[defName="MechaTentacleME"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/verbs</xpath>
 				<value>
 					<verbs>
 						<li Class="CombatExtended.VerbPropertiesCE">
 							<verbClass>CombatExtended.Verb_MeleeAttackCE</verbClass>
 							<defaultCooldownTime>1.2</defaultCooldownTime>
 							<meleeDamageBaseAmount>13</meleeDamageBaseAmount>
 							<meleeDamageDef>Blunt</meleeDamageDef>
-							<meleeArmorPenetration>0.18</meleeArmorPenetration>
+							<meleeArmorPenetration>0.60</meleeArmorPenetration>
```

Comment:
I'm unfamiliar with Hediffs in CE but is `<meleeArmorPenetration>` correct here? Should it be specifying Sharp or Blunt?

Reply:
I'm not entirely familiar with CE hediffs, either.

This new value was submitted by @Magichp via Mediafire. @Magichp, could you explain the rationale for this new value?

Reply:
Yeah, I'm fairly certain this will need to be changed to the updated Sharp and Blunt AP structure.

Reply:
Fixed in 65dd37c 

Assumption: Tentacles are similar blunt armor penetration to the vanilla Power Claw, and no corresponding sharp armor penetration.